collector/user_input_delay_test.go

```diff
@@ -0,0 +1,28 @@
+package collector
+
+import (
+	"reflect"
+	"testing"
+)
+
+func BenchmarkUserInputDelayCollector(b *testing.B) {
+	benchmarkCollector(b, "user_input_delay", NewUserInputCollector)
+}
+
+func TestSplitProcessLabel(t *testing.T) {
+	data := "1:1268 <svchost.exe>"
+	expectedOutput := processLabels{
+		sessionID:   "1",
+		pid:         "1268",
```

Comment:
Quite nice annotation from the test failure here!

Reply:
Very nice! I'll fix this in the next push.